static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -39,6 +41,10 @@ define([
         setProgressTracker(atomId);
         tracking.track('play', getTrackingId(atomId));
 
+        if (document.querySelector('.immersive-main-media .youtube-media-atom')) {
+            document.querySelector('.immersive-main-media').classList.add('atom-playing');
+        }
+
```

Comment:
Just a thought, if there's a youtube atom in the body of the article that starts playing this check will pass when a user plays that video and will add the class to the main media. We should check that the video playing is the main-media video before adding this class. eg. if (closest(player.iframe, '.immersive-main-media__media')) {...}